zerver/models/presence.py

```diff
@@ -30,6 +30,13 @@ class UserPresence(models.Model):
     # queries to fetch all presence data for a given realm.
     realm = models.ForeignKey(Realm, on_delete=CASCADE)
 
+    # The sequence ID within this realm for the last update to this user's presence;
+    # these IDs are generated by the PresenceSequence table and an important part
+    # of how we send incremental presence updates efficiently.
+    # To put it simply, every time we update a UserPresence row in a realm,
+    # the row gets last_update_id equal to that of the previously
+    # updated row + 1.
```

Comment:
"equal to 1 more than the previously updated row in that realm." -- just want to spell out explicitly the realm-specific nature of it, since one could easily imagine the opposite design.